### ISSUE TITLE
Microadenoma

### ISSUE THREAD

**Issue by e-lbtn** [obophenotype/human-phenotype-ontology#7184]
**Preferred term label:**
Microadenoma

**Synonyms**

**Definition (free text, please give PubMed ID)**
A microadenoma is a noncancerous tumor that typically develops in the pituitary gland
PMID: 27974908

**Parent term (use hpo.jax.org/app)**
Pituitary corticotropic cell adenoma (HP:0008291)
Pituitary adenoma (HP:0002893).

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
ORPHA:96253

**Comment by pnrobinson** [comment]
Adding as
Pituitary microadenoma
Def: A pituitary adenoma that is less than 10 mm in diameter.
Comment:Pituitary microadenoma is usually an incidental finding on MRI head. Patients are asymptomatic unless the tumor is hormonally active.
PMID: 32119338
